src/main/kotlin/no/nav/helse/fastsetting/Maksdato.kt

```diff
@@ -26,11 +25,16 @@ fun vurderMaksdato(
             )
             val beregnetMaksdato = maksdato(grunnlag)
 
-            Vurdering.Avklart(
-                    fastsattVerdi = beregnetMaksdato.dato,
-                    fastsattAv = "SPA",
-                    grunnlag = grunnlag,
-                    begrunnelse = beregnetMaksdato.begrunnelse)
+            if (beregnetMaksdato.dato.isBefore(sisteSykepengedag))
+                // § 8-12. Antall sykepengedager er brukt opp eller blir det i sykepengeperioden. Kan være tilfeller der de likevel skal få utbetalt sykepenger (f.eks i deler av perioden)
+                Vurdering.Uavklart(
```

Comment:
Men blir dette egentlig riktig da? For her fastsettes fakta, uavhengig av sisteSykepenge-dag, og maksDato er jo Avklart uavhengig av denne, også må det vel bli opptil behandling.vilkårsvurdering (val vilkårsprøving = sykepengevilkår.evaluer(grunnlag)) å vurdere hvorvidt vilkårene er tilfredsstilt. Hvis Inngangsvilkår svarer at "JA" det er de, så ligger feilen i vilkårsvurderings-koden tenker jeg da :)

ref: val inngangsvilkår = (harOppfyltOpptjeningstid og harOppfyltMedlemskap og harIngenYtelserSomIkkeKanKombineresMedSykepenger
   og erKravetFremsattInnenFrist og tapAvPensjonsgivendeInntektOgMinsteInntekt og _maksAntallSykepengedagerErIkkeBruktOpp_) eller toBeDecided